fiftyone/server/routes/mutation.py

```
@@ -0,0 +1,122 @@
+                            sample[field_name] = label_cls.from_dict(value)
+                        except Exception as e:
+                            errors.append(
+                                f"Failed to parse field '{field_name}': {str(e)}"
+                            )
+                    else:
+                        errors.append(
+                            f"Unsupported label class '{cls_name}' for field '{field_name}'"
+                        )
+                else:
+                    sample[field_name] = value
+            except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)